apps/web/src/pages/DNDDigitalCareersPage/DNDDigitalCareersPage.tsx

```diff
@@ -0,0 +1,533 @@
+          })}
+        </Note>
+        <Heading level="h4" size="h6" className="mb-6">
+          {intl.formatMessage({
+            defaultMessage: "Innovation Corps roles include",
+            id: "3GAO9z",
+            description: "Heading for roles within innovation corps",
+          })}
+        </Heading>
+        <Card className="mb-18 overflow-hidden">
+          <Card.Grid columns={2}>
+            <Card.GridItem>
+              <Heading level="h4" size="h6" className="mt-0">
+                {intl.formatMessage({
+                  defaultMessage: "Information Governance Specialist",
+                  id: "nhWRUg",
+                  description:
+                    "Heading for the information governance specialist role",
+                })}
+              </Heading>
+              <Note className="mb-6">{innovationSalary}</Note>
+              <p>
+                {intl.formatMessage({
+                  defaultMessage:
+                    "Information Governance Specialists in the Innovation Corps support the creation of ontologies and knowledge graphs to ensure information can be effectively shared or retrieved across the organization. They are strong communicators and help disseminate best practices.",
+                  id: "2xDnV4",
+                  description:
+                    "Description of the information governance specialist role",
+                })}
+              </p>
+            </Card.GridItem>
+            <Card.GridItem>
+              <Heading level="h4" size="h6" className="mt-0">
+                {intl.formatMessage({
+                  defaultMessage: "Data Analyst",
+                  id: "0VMNWQ",
+                  description: "Heading for the data analyst role",
+                })}
+              </Heading>
+              <Note className="mb-6">{innovationSalary}</Note>
+              <p>
+                {intl.formatMessage({
+                  defaultMessage:
+                    "Data Analysts in the Innovation Corps will serve as data storytellers in DND. Working closely with clients and stakeholders, they create compelling visuals and narratives that enable others to make better decisions. By providing advice, they also help ensure the ethical and sound use of data.",
+                  id: "Y9Tig+",
+                  description: "Description of the data analyst role",
+                })}
+              </p>
+            </Card.GridItem>
+          </Card.Grid>
+          <Card.Separator className="my-6" />
+          <div className="flex flex-wrap items-center gap-6">
+            <Link
+              mode="solid"
+              href="DSGDigitalUpskilling-GSNPerfectionnementnumerique@forces.gc.ca"
+              external
+            >
+              {intl.formatMessage({
+                defaultMessage: "Contact the Innovation Corps team",
+                id: "BM8p9E",
+                description:
+                  "Link text to start an email to the innovation corps",
```

Comment:
This isn't working as intended, rather than work as an email it is trying to navigate

<img width="1053" height="368" alt="image" src="https://github.com/user-attachments/assets/bb2b0874-bcd0-4ea3-8cdd-667216073fc5" />